### PR TITLE
add go mod and update jwt module path

### PR DESCRIPTION
**Pull request by TamerFlairstech**
While working on [github.com/thermeon/configuration-service](url) I noticed that building fails when trying to import [github.com/thermeon/go-logs](url). After tracking the issue, the reason was that **go-logs** repo imports **sdhook**. And **sdhook** imports [jwt](url) from [github.com/knq/jwt](url) which path was changed by the auther to [github.com/kenshaw/jwt](url).

And therefore, I replaced the old **jwt** module path with the new one in **sdhook**.

I also added `go mod` files.

Also compared this branch the original [github.com/kenshaw/sdhook](url). Didn't find critical updates. And therefore, did not perform merge with latest updates in the original fork.

### CODE REVIEW

**Review by adabour-thermeon — APPROVED** [review]
Well done :+1:

**Review by drunlade — APPROVED** [review]
Good work 👍

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ compatible hook for [Google Stackdriver logging](https://cloud.google.com/loggin
 
 Install in the usual Go way:
 ```sh
-go get -u github.com/knq/sdhook
+go get -u github.com/kenshaw/sdhook
```
Comment (314159) [inline README.md]: Well, since we forked it, shouldn't it say
```suggestion
go get -u github.com/thermeon/sdhook
```

**Review by 314159 — APPROVED** [review]
Approved with one minor change requested.

File: README.md (hunk 1 above)
Comment (TamerFlairstech) [reply README.md]: Thank you for your comment. Done

**Pull request merged**